src/app/tailwind-templates/page.tsx

```diff
@@ -1,8 +1,17 @@
 export default function TailwindTemplates() {
     return (
         <div>
-            <a href="./"><h1 className="font-semibold text-7xl m-4 p-4 text-center">Hello from new-page</h1></a>
-            <h2 className="font-medium text-4xl m-3 p-3 text-center">This is the sub heading...</h2>
+            <a href="./"><h1 className="font-semibold text-7xl m-2 p-2 text-center text-awhite">Hello from new-page</h1></a>
+            <h2 className="font-medium text-5xl m-2 p-2 text-center text-awhite">This is the sub heading...</h2>
+            <p className="w-2/4 font-medium text-lg m-auto p-2 text-center text-awhite">Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut, harum dolorum laborum culpa quis commodi animi sit
+                inventore ex, doloremque nobis itaque eveniet fugiat explicabo? Eligendi sunt obcaecati dicta adipisci? Lorem ipsum
+                dolor sit amet consectetur adipisicing elit. Necessitatibus soluta, ut adipisci dignissimos iusto assumenda est
+                placeat porro quo fugit magnam molestiae nulla suscipit laborum ducimus laboriosam dolores. Illo, temporibus.</p>
+            <div className="flex m-5">
+                <button className="rounded-none bg-adarkgrey m-auto p-3 font-medium text-md text-awhite justify-center">Save Changes</button>
+                <button className="rounded-none bg-ared m-auto p-3 font-medium text-md text-awhite justify-center">Save Changes</button>
+                <button className="rounded-none bg-ablue m-auto p-3 font-medium text-md text-awhite justify-center">Save Changes</button>
+            </div>
```

Comment:
**Question:** How do you envision these templates being used? 
 
Is it a case where we use the documentation and examples as a reference when we need to use them?

Reply:
[App_Pages.pdf](https://github.com/JamesRobertSutcliffe/personality-music-recommender/files/12378969/App_Pages.pdf)

These templates were built to be utilised in the pages that have been posted as issues. We can reference the layout of the page in the attached app_pages PDF. Each Github issue also out lines what content is needed when each page is built. In developing these templates and utilising them when we build pages we will be ensuring all of our elements match, although we can both use out best judgement when is comes to desing of specific pages. 

We can also reference the map of the project user journey which will be useful when getting an idea of how the project will work. 

[Mind_Map.pdf](https://github.com/JamesRobertSutcliffe/personality-music-recommender/files/12379023/Mind_Map.pdf)

Reply:
Sounds good!

### For a future PR [A mind vomit]
If this is the case, we might want to look into creating pre-styled components and embrace the "React way" of handling reusable stuff.

We could have components like these for example: `<Paragraph>`, `<Heading>`, `<SubHeading>`, `<Button background={colour}>`.

By having these templates as components, we can ensure styling and behaviour is consistent across the app. If we were to make changes to how buttons are displayed or behave, we would only have to change the code once in the component - all the other places where the component is used would be updated automatically.

Whereas if we're copying and pasting the multiple styles, it's more troublesome to update things and is prone to mismatch eventually.

Reply:
Noted for future PR!